base_optional_quick_create/model.py

```diff
@@ -38,16 +39,42 @@ def wrapper(cr, uid, name, context=None):
         return wrapper
 
     def _register_hook(self, cr, ids=None):
```

Comment:
@eLBati let's say another module overrides _register_hook for the same class. It's signature does not contain the ids argument, because it is identical to the signature in core Odoo. Your module then calls _register_hook with the new ids argument. If the other module happens to come first, this will cause a crash.